vcpkg.json

```diff
@@ -202,7 +202,7 @@
     },
     {
       "name": "woff2",
-      "version": "1.0.2#4"
+      "version": "1.0.2#5"
```

Comment:
Can we split this into 2 commits? We typically do 1 commit for the baseline update, and subsequent commits for each package upgrade

Reply:
Sure! Done